common/changes/@itwin/presentation-common/presentation-frontend_property_formatting_2023-03-01-10-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-common",
+      "comment": "Added Format lookup in ECSchema for formatting properties with KindOfQuantity",
```

Comment:
should probably mention the new peer dep too

Reply:
Added new change file mentioning new peer dep